paasta_tools/instance/kubernetes.py

```diff
@@ -91,20 +91,22 @@ def set_cr_desired_state(
         num_tail_lines = calculate_tail_lines(verbose)
 
         for pod in pod_list:
-            if num_tail_lines > 0:
-                tail_lines = await get_tail_lines_for_kubernetes_pod(
-                    client, pod, num_tail_lines
+            containers = [
```

Comment:
Any reason this is not a dictionary?

Reply:
It's because in the Swagger spec, I created KubernetesContainer object, which I think is nice for ensuring there is always a name associated with the rest of a container's details (for now its only the tail lines). The list of dictionaries is here just the server-side representation of that object. I don't mind changing it to a dictionary instead though I think it wouldn't be as nice.